src/Engine/Component/Mesh.hpp

```diff
@@ -30,5 +30,21 @@ namespace Component {
              */
             Video::Geometry::Geometry3D* geometry = nullptr;
 
+            /// Highlight selection.
+            /**
+            * @return True if selected else false.
+            */
+            bool GetSelected() const;
+
+            /// Highlight selection.
+            /**
+            * @param Set true if mesh is selected, else false.
```

Comment:
Incorrect doxygen, @param should be followed by parameter name (in this case value) followed by description.